src/io/flutter/sdk/FlutterCommandStartResult.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.sdk;
+
+import com.intellij.execution.ExecutionException;
+import com.intellij.execution.process.OSProcessHandler;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class FlutterCommandStartResult {
+  @NotNull
+  public final FlutterCommandStartResultStatus status;
+
+  @Nullable
+  public final OSProcessHandler processHandler;
+
+  @Nullable
+  public final ExecutionException exception;
+
+  public FlutterCommandStartResult(@NotNull FlutterCommandStartResultStatus status,
```

Comment:
I'd add a few more of these constructors - I note that most call sites pass in null for the last param.

Reply:
Done.